lib/ES/Util.pm

```diff
@@ -191,6 +195,10 @@ sub build_single {
 
     my ( $output, $died );
     if ( $asciidoctor ) {
+        my $dest_xml = $index->basename;
```

Comment:
This was just wrong before. It happened to work because we only built books that started at a file named `index.asciidoc`.